ethpm_types/contract_type.py

```diff
@@ -129,6 +129,55 @@ def extract_sourcemap_item(expanded_row, item_idx, previous_val=None):
             yield item
 
 
+class ABIList(list):
+    """
+    Adds selection by name, selector and keccak(selector).
+    """
+
+    def __init__(
+        self,
+        iterable=(),
+        *,
+        selector_size=32,
+        selector_hash: Optional[Callable[[str], bytes]] = None,
+    ):
+        self._selector_size = selector_size
+        self._selector_hash = selector_hash
+        super().__init__(iterable)
+
+    def __getitem__(self, item: Union[str, bytes, MethodABI, EventABI]):  # type: ignore
```

Comment:
Should this handle int and slice too?

Reply:
they are handled by the `super().__getitem__(item)` on the last line